tfmigrate-apply/action.yaml

```diff
@@ -9,17 +9,24 @@ inputs:
     description: 'GitHub Access Token'
     required: false
     default: ${{ github.token }}
+
 runs:
   using: composite
   steps:
     - uses: suzuki-shunsuke/tfaction/get-global-config@main
       id: global-config
     - uses: suzuki-shunsuke/tfaction/get-target-config@main
       id: target-config
+    - uses: suzuki-shunsuke/tfaction/get-global-config@main
+      id: global-config
```

Comment:
`global-config` is duplicated.

ref:

* https://github.com/suzuki-shunsuke/tfaction/issues/906